appnav/src/main/kotlin/io/element/android/appnav/LoggedInFlowNode.kt

```diff
@@ -192,6 +195,12 @@ class LoggedInFlowNode @AssistedInject constructor(
                 matrixClient.sessionVerificationService().setListener(verificationListener)
                 mediaPreviewConfigMigration()
 
+                sessionCoroutineScope.launch {
+                    // Wait for the network to be connected before pre-fetching the max file upload size
+                    networkMonitor.connectivity.first { networkStatus -> networkStatus == NetworkStatus.Connected }
+                    matrixClient.getMaxFileUploadSize()
```

Comment:
This is to try to ensure we have this value ahead of time, even if we need to load it asynchronously later.